t/15_qesap_azure.t

```diff
@@ -105,16 +105,16 @@ subtest '[qesap_az_vnet_peering_delete]' => sub {
     my $qesap = Test::MockModule->new('qesapdeployment', no_auto => 1);
     my @calls;
     $qesap->redefine(get_current_job_id => sub { return 42; });
-    $qesap->redefine(script_output => sub { push @calls, $_[0]; return 'GYROS'; });
+    $qesap->redefine(script_output => sub { push @calls, $_[0]; return 'NEMO'; });
```

Comment:
NEMO > GYROS? :D

Reply:
was not following the theme of the unit testing file.